R/launcher-functions.R

```diff
@@ -295,52 +295,56 @@ launcherSubmitJob <- function(name,
                   applyConfigSettings = applyConfigSettings)
 }
 
+#' Execute an R Script as a Launcher Job
+#'
+#' Convenience function for running an R script as a launcher job using whichever
+#' R is found on the path in the launcher cluster.
+#'
+#' See [launcherSubmitJob()] for running jobs with full control over command,
+#' environment, and so forth.
+#'
+#' @param script Fully qualified path of R script. Must be a path that is
```

Comment:
The documentation feels a bit confusing to me. This should be the path to an R script on the user's local machine, is that right? Why does that same path need to exist on the cluster? (E.g. couldn't we just form a 'safe' version of the path for running with the launcher if necessary)?

Odds are I'm just misunderstanding some of the semantics here.

Reply:
Does this function handle automagically uploading the R script to be run to the cluster?

Reply:
This function runs an R script in the same way that the Jobs pane does. It does not upload the script, it requires the script’s location being mounted in the cluster. The pane also uses whichever R is on the path in the cluster/container. Supporting specification of other R versions is not something we are doing for 1.2, but probably for 1.2.x.

Reply:
This function runs an R script in the same way that the Jobs pane does. It does not upload the script, it requires the script’s location being mounted in the cluster. The pane also uses whichever R is on the path in the cluster/container. Supporting specification of other R versions is not something we are doing for 1.2, but probably for 1.2.x.